styles/arch-wiki/catppuccin.user.css

```diff
@@ -254,7 +261,7 @@
     
     .mw-parser-output a.external, .mw-parser-output a.extiw {
         background-image: none !important;
-        padding-right: 1px !important;
+        padding-right: 0px !important;
```

Comment:
lets not change the padding

Reply:
without that theres gonna be a small whitespace right next to every link though

Reply:
thats not our choice to make, thats up to archlinux. raise the issue with them

Reply:
> thats not our choice to make, thats up to archlinux. raise the issue with them

alright i'll just remove the padding ig